simulator/source/Combat_simulator.cpp

```diff
@@ -452,6 +454,39 @@ void Combat_simulator::mortal_strike(Sim_state& state)
     logger_.print("Current rage: ", int(rage));
 }
 
+void Combat_simulator::devastate(Sim_state& state)
+{
+    //TODO: Add DPR support
+    logger_.print("Devastate!");
+    int bonus = (config.exposed_armor && apply_delayed_armor_reduction) ? 0 : std::vector<int>{0, 35*2, 35*3, 35*4, 35*5, 35*5}[std::min(sunder_armor_stacks_, 5)];
```

Comment:
Done. the std::vector structure was just borrowed from a similar case of non-linear scaling with an integer (improved execute rage cost reduction) so I just figured that should be done the same way here as well. I really appreciate suggestions like using the assert instead of the min approach here though, I suck at stylistic things like that.